crowdsourcing/wpconvlib/src/conversation.test.ts

```diff
@@ -148,32 +153,68 @@ export class ConversationTest {
       .beFalse();
   }
 
+  public "Map of structureConversaton with deleted comments"() {
+    const rootComment =
+      conversation.structureConversaton(
+        JSON.parse(JSON.stringify(example_conversation5)));
+    const commentIds: string[] = [];
+    should(rootComment).beOkay();
+    should(rootComment!.isRoot).equal(true);
```

Comment:
Generally it's not good to use !., it's better to use proper null checking instead so the compiler doesn't complain here. Look into enabling strict null checks for the project.